packages/secure_storage/amplify_secure_storage/android/build.gradle

```
@@ -28,20 +28,20 @@ android {
     compileSdkVersion 33
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
     }
 
     kotlinOptions {
-        jvmTarget = '1.8'
+        jvmTarget = '17'
     }
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'
     }
 
     defaultConfig {
-        minSdkVersion 23
+        minSdkVersion 24
```

This package supports SDK 23. There are tests that run for 23 which are failing with this change. 

```suggestion
        minSdkVersion 23
```

---

Since this package is internal and we only support sdk 24+ in Amplify you can update the tests to run for sdk 24 instead.

https://github.com/aws-amplify/amplify-flutter/blob/main/packages/secure_storage/amplify_secure_storage/android/src/test/kotlin/com/amazonaws/amplify/amplify_secure_storage/EncryptedKeyValueRepositoryTest.kt#L19

---

I pushed a commit to update the tests